indico/web/templates/_protection_messages.html

```diff
@@ -25,9 +25,16 @@
                     {% trans %}Protected{% endtrans %}
                 </div>
                 <div>
-                    {% trans -%}
-                        This object is <strong>only</strong> accessible by the <strong>users specified</strong> above and the <strong>managers</strong> of <strong>parent resources</strong>.
-                    {%- endtrans %}
+                    {% if users_allowed %}
+                        {% trans -%}
+                            This object is <strong>only</strong> accessible by the <strong>users specified</strong> above and the <strong>managers</strong> of <strong>parent resources</strong>.
```

Comment:
Let's wrap this? :D 

I think the overly long line is still from the time where we did not collapse whitespace inside trans blocks...